gef.py

```diff
@@ -10241,6 +10261,32 @@ def __parse_gdb_info_sections(self) -> Generator[Section, None, None]:
                 continue
         return
 
+    def __parse_info_mem(self) -> Generator[Section, None, None]:
+        """Get the memory mapping from GDB's command `monitor info mem`"""
+        stream = StringIO(gdb.execute("monitor info mem", to_string=True))
+        for line in stream:
+            if not line:
+                break
+            try:
+                # FORMAT
+                # ffffff2a65e0b000-ffffff2a65e0c000 0000000000001000 -r-
+                # ^--- start       ^--- end         ^--- offset      ^^^-- perms
+                # `perms` are `urw`, for "usermode", "readable", "writable"
```

Comment:
Not convinced of the value of the comment here: probably better in a CI test for this function.

```suggestion
```